src/imagelightbox.js

```diff
@@ -550,6 +554,9 @@
         $(window).on('resize.ilb7', _setImage);
 
         $(document).ready(function() {
+            $(this).on("start.ilb7", function (e,b) {
+                console.log(b);
```

Comment:
Nothing at the moment; just proof of concept. This is needed for implementation of history API. The `console.log(b)` can be removed.